src/commands/services/list.ts

```diff
@@ -286,15 +303,27 @@ export default class ServicesList extends Command {
           },
           phone: {
             header: "phone",
-            get: row => row.serviceMetadata && row.serviceMetadata.phone
+            get: row =>
+              row.serviceMetadata &&
+              (row.serviceMetadata.phone === undefined
+                ? "undefined"
+                : row.serviceMetadata.phone.split('"').join(""))
```

Comment:
```suggestion
                : row.serviceMetadata.phone.replace('"', ""))
```
If I understood the scope, this should remove the `"` from that string.
If yes, could the suggestion valid?

In any cases what do you think to create a function since this code is repeated few times:
```typescript
const rowRpr = (value: string | undefined) => {
        if(value === undefined){
          return "undefined";
        }
        return value.replace('"', "");
      }
```